collectors/idlejitter.plugin/metadata.yaml

```diff
@@ -2,31 +2,38 @@ meta:
   plugin_name: idlejitter.plugin
   module_name: idlejitter.plugin
   monitored_instance:
-    name: IdleJitter
+    name: Idle OS Jitter
     link: ''
     categories:
-    - data-collection.synthetic-checks
+      - data-collection.synthetic-checks
     icon_filename: 'syslog.png'
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
     description: ''
-  keywords: []
+  keywords:
+    - latency
+    - jitter
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Monitor IdleJitter performance for optimal system idle process operations. Monitor CPU idle times, wake-ups, and power states to optimize system resource usage.'
-    method_description: ''
+    metrics_description: >
+      Monitor delays in timing for user processes caused by scheduling limitations to optimize the system to run
+      latency sensitive applications with minimal jitter, improving consistency and `quality of service.
```

Comment:
Austin, is there a backtick missing here?

Reply:
Actually, there’s not supposed to be a backtick there at all.

Reply:
Should be resolved with the latest commit.